### PR TITLE
feat: add vite-aliases

### PR DESCRIPTION
**Pull request by Subwaytime**
### `General`

> ✏️ Mark the necessary items without changing the structure of the PR template.

- [x] Pull request template structure not broken

### `Type`

> ℹ️  What types of changes does your code introduce?

> 👉 _Put an `x` in the boxes that apply_

- [ ] Fix
- [x] Feature 

### `Checklist`

> ℹ️  Check all checkboxes - this will indicate that you have done everything in accordance with the rules in [CONTRIBUTING](https://github.com/vuejs/awesome-vue/blob/master/.github/contributing.md).

> 👉  _Put an `x` in the boxes that apply._

- [x] Title as described.
- [x] Make sure you put things in the right category!
- [x] Always add your items to the end of a list.
- [x] Your plugin/tool is working with Vite 2.0 and onward.

#### `Open Source`

- [x] Link description does not contain a link to an author / third-party resource
- [x] The documentation (README) contains a description of the project, illustration of the project with a demo or screenshots and a CONTRIBUTING section
- [x] The documentation is in English.
- [x] The project is active and maintained.
- [x] The project accepts contributions.
- [x] Not a commercial product

#### `Apps/Websites`

- [x] The website is available without errors or ssl certificate problems, and load in a reasonable amount of time.
- [ ] The website is using Vite intensively.
- [ ] The website is original and not too simple. For that reason, blogs and simple landing pages are rejected.
- [ ] A commercial product using Vite, provided that guests could reasonably check out how Vite was used A commercial product using Vite, provided that guests could reasonably check out how Vite was used (i.e. A headless CMS which uses Vite for the Admin/editor Area and offers a free tier).

### CODE REVIEW

**Comment by Subwaytime** [comment]
Atm, there are Issues with MacOs and Unix (Linux). 
Till this is resolved on `vite-aliases` Site, it wont work perfectly with Vite 2.

**Comment by Subwaytime** [comment]
Next Version of `vite-aliases` will include MacOs and Unix Support with Vite 2.
Release will be later today!

**Comment by antfu** [comment]
Is it ready now?

**Comment by Subwaytime** [comment]
Yes! :)

**Comment by antfu** [comment]
I think we can move this to the "framework-agnostic" section.

**Comment by Subwaytime** [comment]
"framework-agnostic" --> helpers ?

**Comment by Subwaytime** [comment]
Done!

**Comment by antfu** [comment]
Can you fix the merging conflicts? Thanks

**Comment by Subwaytime** [comment]
Should be resolved.. i hope atleast ^^

File: README.md [hunk 1]
```diff
@@ -129,8 +129,15 @@ A curated list of awesome things related to <a href='https://github.com/vitejs/v
 - [vite-plugin-fonts](https://github.com/stafyniaksacha/vite-plugin-fonts) - Webfont loader.
 - [vite-imagetools](https://github.com/JonasKruckenberg/vite-imagetools) - Load and transform images using url query parameters.
 
+
+### Helpers
+- [vite-aliases](https://github.com/subwaytime/vite-aliases) - Alias auto generation based on Projectstructure.
```
Comment (antfu) [inline README.md:134]: Should move to L154

**Review by antfu — CHANGES_REQUESTED** [review]
Please see the comments

**Pull request closed without merging**